src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java

```diff
@@ -108,143 +88,20 @@ public String sendOrRespondMessage(final String profile, final Integer projectId
       if (CollectionUtils.isEmpty(nonOCDS.getReceiverList())) {
         throw new JaggaerRPAException("Suppliers are mandatory if broadcast is 'No'");
       }
-      var supplierString = validateSuppliers(procurementEvent, nonOCDS);
+      var suppliers = rpaGenericService.getValidSuppliers(procurementEvent,
+          nonOCDS.getReceiverList().stream().map(org -> org.getId()).collect(Collectors.toList()));
```

Comment:
Sonar suggests replacing this with a method reference (i.e. `OrganizationReference1::getId`)